comps/dataprep/src/opea_dataprep_microservice.py

```
@@ -66,9 +67,28 @@ async def ingest_files(
 
     try:
         # Use the loader to invoke the component
-        response = await loader.ingest_files(
-            files, link_list, chunk_size, chunk_overlap, process_table, table_strategy, ingest_from_graphDB
-        )
+        if dataprep_component_name == "OPEA_DATAPREP_REDIS":
+            response = await loader.ingest_files(
+                files,
+                link_list,
+                chunk_size,
+                chunk_overlap,
+                process_table,
+                table_strategy,
+                ingest_from_graphDB,
+                index_name,
+            )
+        else:
+            if index_name:
+                logger.error(
```

Why should we raise an exception here for user passing the `index_name`?
Looks like this parameter is not passed to `loader.ingest_files` in line 89.

---

so far it is only supporting OPEA_DATAPREP_REDIS, in case index_name assigned with other DATAPREP_COMPONENT_NAME. it returns error msg with explanation.